test/lib/RuleManager.test.ts

```diff
@@ -257,7 +257,7 @@ describe('RuleManager', () => {
 						parsedRes = JSON.parse(results);
 					}
 					expect(parsedRes).to.be.an("array").that.has.length(1);
-					Sinon.assert.callCount(uxSpy, 0);
+					Sinon.assert.callCount(uxSpy, 1);
```

Comment:
None of these calls have the `--projectdir`, so they're all trying to log the warning about skipping SFGE. Hence, the incrementation by one.